src/Captcha/index.jsx

```diff
@@ -0,0 +1,14 @@
+import { Component } from 'react';
+import { observer } from 'mobx-react';
+
+/**
+ * This component is necessary for the recaptcha element to appear.
+ */
+@observer
```

Comment:
Why is it an observer? It doesn't seem like this component does anything with observable props.

```suggestion
```

Reply:
Agreed